fastapp/fastapp/ui/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" hidden>
+  <head>
+    <script src="https://use.fontawesome.com/3a2eaf6206.js"></script>
+    <script type="module" src="https://cdn.skypack.dev/twind/shim"></script>
+  </head>
+  <body class="bg-gray-100">
+    <div class="flex items-center justify-center h-screen">
+      <div class="overflow-hidden rounded-lg bg-white shadow w-96">
+        <div class="border-b border-gray-200 bg-white px-4 py-4 sm:px-6">
+          <h3 class="text-lg font-medium leading-6 text-gray-900">
+            Upload Hyperpack
+          </h3>
+        </div>
+
+        <div class="px-4 py-4">
+          <form
+            action="/upload-hyperpack"
+            enctype="multipart/form-data"
+            method="post"
+          >
+            <div class="flex justify-center">
+              <div>
+                <label for="inputFile" class="cursor-pointer">
+                  <div
+                    class="border rounded-lg mb-4 w-48 shadow-sm text-center"
+                  >
+                    Select Hyperpack
+                    <div class="flex justify-center mt-4 mb-4">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        fill="none"
+                        viewBox="0 0 24 24"
+                        stroke-width="1.5"
+                        stroke="currentColor"
+                        class="w-12 h-12"
+                      >
+                        <path
+                          stroke-linecap="round"
+                          stroke-linejoin="round"
+                          d="M7.5 7.5h-.75A2.25 2.25 0 004.5 9.75v7.5a2.25 2.25 0 002.25 2.25h7.5a2.25 2.25 0 002.25-2.25v-7.5a2.25 2.25 0 00-2.25-2.25h-.75m0-3l-3-3m0 0l-3 3m3-3v11.25m6-2.25h.75a2.25 2.25 0 012.25 2.25v7.5a2.25 2.25 0 01-2.25 2.25h-7.5a2.25 2.25 0 01-2.25-2.25v-.75"
+                        />
+                      </svg>
+
+                      <input
+                        id="inputFile"
+                        name="file"
+                        type="file"
```

Comment:
Since we're looking to only upload one hyperpack at a time, let's enforce that restriction within the `<input />`:
```html
<input 
  id="inputFile"
  name="file
  type="file" 
  multiple="false"
  accept=".hyperpack"
/>
```
While _only_ allowing uploads of `.hyperpackage` files.

Reply:
Good call; I'll restrict it to a single zip